scripts/build-multiarch-baseimg.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Yes, we can. I've removed this script and incorporated the code for `multi-arch` base image creation into the Makefile. Since the `multi-arch` images already include the `amd64` version, I kept only **one** target for creating the `multi-arch` base image in the Makefile and made relevant changes in other scripts.